File: Java/length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        String str = s.trim();
        if (str.length() == 0) {
            return 0;
        }
        int length = 0;
        char[] chars = str.toCharArray();
        for (int i = chars.length - 1; i >= 0; i--) {
            if (chars[i] == ' ') {
                break;
            } else {
                length++;
            }
        }
        return length;
    }
}

File: Kotlin/length-of-last-word.kt

class Solution {
	fun lengthOfLastWord(s: String): Int {
    	val str = s.trim()
    	if (str.length == 0) return 0
    	try {
        	return (str.length - 1 - str.indexOfLast { it == ' ' })
    	} catch(e: Exception) {
        	return str.length
    	}
	}
}
